### PR TITLE
add kill-executor handler

### PR DESCRIPTION
**Pull request by huijunwu**
From #1770

Add kill-executor handler for shell.
We consider it as a 'write' operation for the container, which is different from the existing 'read' handlers.
The security check will be added in another PR.

### CODE REVIEW

File: heron/shell/src/python/handlers/killexecutorhandler.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  def post(self):
+    """ post method """
+    logger = logging.getLogger(__file__)
+    logger.info("Received 'Killing parent executor' request")
+    # todo(huijun): add security check
+    logger.info("Killing parent executor response 200")
+    self.set_status(200)
+    self.finish()
+    logger.info("Killing parent executor")
+    os.killpg(os.getppid(), signal.SIGTERM)
```
Comment (objmagic) [inline heron/shell/src/python/handlers/killexecutorhandler.py]: Add newline at end of the file.

**Comment by objmagic** [comment]
Also, have you tried locally that it will work?

File: heron/shell/src/python/handlers/killexecutorhandler.py (hunk 1 above)
Comment (huijunwu) [inline heron/shell/src/python/handlers/killexecutorhandler.py]: done

**Comment by huijunw** [comment]
tested on local machine

**Comment by objmagic** [comment]
@huijunw don't forget to PR twitter internal repo

**Pull request merged**